blaze/compute/tests/test_pandas_compute.py

```diff
@@ -680,20 +697,23 @@ def test_str_lower():
     assert_series_equal(expected, result)
 
 
-class TestStrCat():
-    ds = dshape('3 * {name: string[10], comment: string[25], num: int32}')
-    s = symbol('s', dshape=ds)
-    data = [('alice', 'this is good', 0), ('suri', 'this is not good', 1),
-            ('jinka', 'this is ok', 2)]
-    df = pd.DataFrame(data, columns=['name', 'comment', 'num'])
+def test_str_cat():
+    res = compute(tbig.name.str_cat(tbig.sex), dfbig)
```

Comment:
Use `dfbig` existing dataframe for tests. This includes a variable length and fixed length string in the dshape:
`{name: string, sex: string[1], amount: int, id: int}`